pkgs/applications/networking/browsers/tor-browser/default.nix

```diff
@@ -99,8 +99,8 @@ lib.warnIf (useHardenedMalloc != null)
   sources = {
     x86_64-linux = fetchurl {
       urls = [
-        "https://dist.torproject.org/torbrowser/${version}/tor-browser-linux64-${version}_${lang}.tar.xz"
         "https://archive.torproject.org/tor-package-archive/torbrowser/${version}/tor-browser-linux64-${version}_${lang}.tar.xz"
+        "https://dist.torproject.org/torbrowser/${version}/tor-browser-linux64-${version}_${lang}.tar.xz"
```

Comment:
I'm not sure if it is nice of us to hit archive.torproject.org first. It might be a slower machine that gets easier overloaded than the official "primary" machine for distribution. On the flip side, dist.torproject.org removes old releases, so if an old Tor Browser is to be built, it initially fails and tries again with the archive. I would prefer that over requesting everything directly from the archive.

Reply:
Well, dist only ever has the latest release. Also 13.0 was released on archive before dist.
Additionally I talked with a Tor Browser dev recently (regarding the Mullvad Browser releases) and got the following statement:
> More generally, you should not use [dist.torproject.org](http://dist.torproject.org/) in automation since we only typically keep
the latest build on that server, whereas [archive.torproject.org](http://archive.torproject.org/) URLs are stable and should be
available forever into the future.